organizations/user.go

```diff
@@ -171,3 +171,70 @@ func UpdateProfilePicture(w http.ResponseWriter, r *http.Request) {
 	utils.GetSuccess("Profile picture updated", result, w)
 
 }
+
+// Delete single member from an organizatin
+func DeleteMember(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	member_collection, org_collection := "members", "organizations"
+	orgId := mux.Vars(r)["id"]
+	memberId := mux.Vars(r)["mem_id"]
+
+	pOrgId, err := primitive.ObjectIDFromHex(orgId)
+	if err != nil {
+		utils.GetError(errors.New("invalid id"), http.StatusBadRequest, w)
+		return
+	}
+
+	// get organization
+	orgDoc, _ := utils.GetMongoDbDoc(org_collection, bson.M{"_id": pOrgId})
+	if orgDoc == nil {
+		fmt.Printf("org with id %s doesn't exist!", orgId)
+		utils.GetError(errors.New("operation failed"), http.StatusBadRequest, w)
+		return
+	}
+
+	checkMember, err := utils.GetMongoDbDoc(member_collection, bson.M{"_id": memberId})
```

Comment:
The _id requires a primitive value. memberId used here isn't a primitive. This isn't right. It should be just like you did when trying to get organization. Convert the memberId to primitive. Then use that to do the query